widget/ui/src/components/Collapse/Collapse.tsx

```diff
@@ -0,0 +1,37 @@
+import type { PropTypes } from './Collapse.types';
+import type { PropsWithChildren } from 'react';
+
+import React, { useRef } from 'react';
+
+import { ChevronDownIcon, ChevronUpIcon } from '../../icons';
+import { Divider } from '../Divider';
+
+import { Container, Content } from './Collapse.styles';
+
+export function Collapse(props: PropsWithChildren<PropTypes>) {
+  const { children, expanded, onClick } = props;
+
+  const content = useRef<HTMLDivElement>(null);
+
+  return (
+    <Container onClick={onClick}>
+      {children}
+
+      <Content
+        ref={content}
+        style={{
+          maxHeight: !expanded ? 0 : content?.current?.scrollHeight,
+        }}
+        className="accordion__content">
+        <Divider size={12} />
+        {props.content}
+      </Content>
+      <Divider size={12} />
+      {expanded ? (
+        <ChevronUpIcon color={'black'} size={16} />
+      ) : (
+        <ChevronDownIcon color={'black'} size={16} />
+      )}
```

Comment:
For general use case it is better to remove the icon and style from the collapse